pkgs/development/python-modules/sphinx-multiversion/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, sphinx
+, unittestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-multiversion";
+  version = "0.2.4";
+
```

Comment:
Can we set pyproject = true or at least format = "setuptools";?

Reply:
I can specify the format. I can't use `pyproject` because the project doesn't actually specify build in it.

Reply:
Did you try it? pyproject fallsback to support setup.cfg/py IIRC

Reply:
Yes, I have tried it, but the fallback works probably only if setuptools is explicitly included as a build dependency. That wasn't clear initially because I was unaware it should work. The documentation also gave me the idea that I should not add `setuptools` because that would include the legacy setuptools hooks. Is this the correct approach now?

Reply:
@SuperSandro2000 ping on this please?

Reply:
Yeah, when switching to pyproject = true you need to explictily include setuptools. We want to avoid drawin in implicit things as that caused sneaky setuptools dependencies in many places and breaks things when we fix those small leakages.

so you probably need to set the following:

```
pyproject = true;

build-system = [ setuptools ];
```

Reply:
It is done. @SuperSandro2000 it should be ready for review.

Reply:
@SuperSandro2000  ping on this?